packages/plugin-ethereum/src/EthereumChildPlugin.ts

```diff
@@ -47,10 +49,19 @@ export class EthereumChildPlugin extends ChildPlugin<EthereumChildPluginConfig>
           this.logger("middleware error: ", err);
         });
     } else if (action.type === "PARENT_REQUEST" && action.data.type === "WEB3_ENABLE") {
-      this.dispatch({
-        type: REQUEST_VIEW_ACTION,
-        payload: { route: "/ethereum/web3enable", requestID: action.requestID },
-      });
+      if (this.config.burnerPreference === "always") {
+        const burnerProvider = new BurnerProvider({
+          rpcUrl: this.config.rpcURL, // oof I don't like the difference in caps here rpcUrl is the standard
```

Comment:
Microsoft agrees with you: >2 characters should be camel cased. I generally use 3 as the cutoff, but we should follow the standard. will do this in a subsequent PR
https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)